api/src/pcapi/routes/native/v1/serialization/bookings.py

```diff
@@ -87,6 +87,14 @@ class Config:
         orm_mode = True
 
 
+class ExternalBookingResponse(BaseModel):
+    class Config:
+        orm_mode = True
+
+    barcode: str
+    seat: Optional[str]
+
```

Comment:
Détail: Pour les serializers d'habitude on déclare d'abord les attributs puis la class de config, c'est mieux si on garde le même style.
```suggestion
class ExternalBookingResponse(BaseModel):
    barcode: str
    seat: Optional[str]
    
    class Config:
        orm_mode = True
```